### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by mirouhml**
- Adjust sections for desktop.
- Adjust menu for desktop.
- Add media query.
- Add functional desktop menu.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -14,63 +14,112 @@
...
-                    <li id="lang1">css</li>
-                    <li id="lang2">html</li>
-                    <li id="lang3">bootstrap</li>
-                    <li id="lang4">Ruby</li>
-                </ul>
-                <button class="featured-project-button" type="button">See Project</button>
             </div>
             <ul class="projects-container">
-                <li id="project1">
+                <li class="project">
```
Comment (ahmetbozaci) [inline index.html]: - [x] The first project view is different than others on the desktop version. For stick to the design please change it.

### ORIGINAL
image

___
### YOUR DESIGN
image

File: index.html [hunk 2]
```diff
@@ -140,19 +189,21 @@ <h2 class="project-title">Profesional Art Printing Data</h2>
                             <li>Ruby</li>
                         </ul>
                     </div>
-                    <button class="project-button" type="button">See Project</button>
+                    <div class="project-card-hover-content">
+                        <button class="project-button" type="button">See Project</button>
```
Comment (ahmetbozaci) [inline index.html]: - [x] The button should take you to the Live Demo of the featured project or to the GitHub repository. Please apply it to all project buttons.

File: index.html [hunk 3]
```diff
@@ -192,31 +243,35 @@ <h2>Skills</h2>
                 </li>
             </ul>
         </section>
-        <section class="contact-me">
+        <section class="contact-me" id="contact-me">
```
Comment (ahmetbozaci) [inline index.html:246]: - [x] Also bold text(I'm always interested ...) now shares the same row with the form. In your design, it looks like the center. 
**_Please fix it._**

ORIGINAL
image
____
YOUR DESIGN
image

File: index.html [hunk 4]
```diff
@@ -14,63 +14,112 @@
...
+                            <img src="src/twitter.svg" alt="Twitter icon">
                         </a>
                     </li>
                 </ul>
             </div>
         </section>
-        <section class="works-section">
+        <section class="works-section" id="portfolio">
+            <div class="works-section-title">
+                <h1 id="works-section-title">My Recent Works</h1>
```
Comment (ahmetbozaci) [inline index.html]: - [x] You have already one h1 tag on the page. The page should include only one [h1 tag](https://www.w3schools.com/tags/tag_hn.asp#:~:text=Note%3A%20Only%20use%20one%20%3Ch1%3E%20per%20page%20%2D%20this%20should%20represent%20the%20main%20heading/subject%20for%20the%20whole%20page.%20Also%2C%20do%20not%20skip%20heading%20levels%20%2D%20start%20with%20%3Ch1%3E%2C%20then%20use%20%3Ch2%3E%2C%20and%20so%20on.). Please change to another hx tag.

File: index.html [hunk 5]
```diff
@@ -192,31 +243,35 @@ <h2>Skills</h2>
...
+                            </li>
+                            <li>
+                                <textarea class="input-style" id="msg" required="required" name="user_message" placeholder="Enter text here" maxlength="500"></textarea>
+                            </li>
+                            <li>
+                                <button type="submit" class="featured-project-button">
+                                    Get in touch
+                                </button>
+                            </li>
+                        </ul>
```
Comment (ahmetbozaci) [inline index.html:274]: ### OPTIONAL

- [x] Footer icons should be centered. Also, 5 icons are included in the original design. To stick to the design you can fix it.

image

File: index.html [hunk 6]
```diff
@@ -14,63 +14,112 @@
...
+                <li>
+                    <a href="https://www.linkedin.com/in/ammar-hamlaoui-514909189/">
+                        <img src="src/linkedin.svg" alt="Linkedin icon">
+                    </a>
+                </li>
+                <li>
+                    <a href="https://twitter.com/kuronomirou">
+                        <img src="src/twitter.svg" alt="Twitter icon">
+                    </a>
+                </li>
```
Comment (ahmetbozaci) [inline index.html:69]: ### OPTIONAL
- [x] 5 icons are included in the original design. To stick to the design you can fix it
### ORIGINAL
image

File: styles.css [hunk 7]
```diff
@@ -461,3 +491,265 @@ textarea {
...
+
+  #separator2 {
+    border-bottom: 1px solid #fff;
+    width: 1156px;
+    margin: auto;
+    margin-bottom: 146px;
+  }
+
+  ul#information-list {
+    display: flex;
```
Comment (ahmetbozaci) [inline styles.css:706]: ### OPTIONAL

- [x] For DRY CSS you should combine duplicate properties. Example, you can create class `.d-flex { display: flex}` and use this class in HTML file.Flex is just an example you can do it to duplicate properties.



- [More information about DRY principles](https://www.freecodecamp.org/news/7-important-tips-for-writing-better-css/) 

- [Html and css best practices in curriculum](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)

File: styles.css [hunk 8]
```diff
@@ -461,3 +491,265 @@ textarea {
...
+    flex-direction: row;
+    justify-content: space-between;
+    text-align: left;
+    margin-bottom: 131px;
+    margin-right: 141px;
+    margin-left: 141px;
```
Comment (ahmetbozaci) [inline styles.css]: ### OPTIONAL

- [x] You can write them in one line. `margin: top right bottom left`
[DRY CSS](https://www.freecodecamp.org/news/7-important-tips-for-writing-better-css/#:~:text=Use%20Shorthands,look%20like%20this%3A)

File: index.html [hunk 9]
```diff
@@ -140,19 +189,21 @@ <h2 class="project-title">Profesional Art Printing Data</h2>
...
-                <h1 id="about-me-title">About me</h1>
+                <h2 id="about-me-title">About me</h2>
                 <p id="about-me-description">Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
-                <button class="featured-project-button" type="button">Get My Resume</button>
+                <button class="about-me-button" type="button">Get My Resume</button>
             </div>
             <div id="separator2"></div>
             <ul id="information-list">
-                <li>
+                <li class="information-list-titles">
```
Comment (ahmetbozaci) [inline index.html:206]: - [x] In the about me, section languages, framework, and skills part is not centered. Check how is in original design and please fix it.

### YOUR DESIGN
image
__
### ORIGINAL
image

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# STATUS: Changes Required  ♻️

### Hello @mirouhml  ✋🏼 
#### Your project is great. You are almost done. However, there are some issues you should fix. Please fix them then your project will be approved.

___ 

### 💯  Nice to have 💯
- Your git-flow is correct. ✔️
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️
- You know how to use media queries✔️
___


### ♻️ Request changes ♻️

- [ ] _Check the comments under the review._

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
___

### Cheers and Happy coding! 💻 👏

almost

___

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_


_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mentioning me as [ahmetbozaci](https://github.com/ahmetbozaci), I will receive a notification and will respond._
___
[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

File: index.html [hunk 10]
```diff
@@ -80,9 +129,9 @@ <h2 class="project-title">Profesional Art Printing Data</h2>
                             <li>Ruby</li>
                         </ul>
                     </div>
-                    <button class="project-button" type="button">See Project</button>
+                    <button onclick="location.href='https://mirouhml.github.io/portfolio/index.html'" class="project-button" type="button" id="first-project-button">See Project</button>
```
Comment (mateo951) [inline index.html:132]: Addressed 🔥🔥 Link is working as expected

File: index.html (hunk 3 above)
Comment (mateo951) [reply index.html:246]: Addressed 😎

File: index.html [hunk 11]
```diff
@@ -14,63 +14,112 @@
...
         </section>
-        <section class="works-section">
+        <section class="works-section" id="portfolio">
+            <div class="works-section-title">
+                <h2 id="works-section-title">My Recent Works</h2>
+                <div id="separator"></div>
+            </div>
             <div id="featured-project">
-                <div id="works-section-title">
-                    <h1>My Recent Works</h1>
```
Comment (mateo951) [inline index.html:58]: Nice work removing the extra `h1` tags ✔

File: index.html (hunk 5 above)
Comment (mateo951) [reply index.html:274]: Fixed 👌🚀

**Review by mateo951 — APPROVED** [review]
Hi @mirouhml, 

Exceptional work making the changes requested by the previous reviewer 🙌
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**